test/NuGet.Server.V2.Tests/NuGetODataControllerTests.cs

```diff
@@ -238,73 +238,6 @@ public void FindPackagesByIdDoesNotHitBackendWhenIdIsEmpty()
             }
         }
 
-        public class SearchMethod
```

Comment:
Re-added in my last commit and configured them to use "real" repository against temporary directory.
But the tests stil fail for some inputvalues.
The tests expect only the latest version of each package to be returned, but the repository (and subsequent the controller) returns all versions of packages matching searchterms.
I checked this against the old NuGet.Server and it seems like my code behaves the same way as the new version.
Do you think the tests (and MyGetGallery implementation) might be wrong? Or should we try to filter to only latest version?